app/views/Import.js

```diff
@@ -86,15 +90,15 @@ const ImportScreen = props => {
             }
             style={styles.buttonTouchable}>
             <Typography style={styles.buttonText}>
-              {languages.t('label.import_takeout').toUpperCase()}
+              {languages.t('import.google.visit_button_text')}
```

Comment:
Was this intentionally made not capitalized

Reply:
Yep, because our buttons styles should be determined at the button level not at the consumer level.

Reply:
<img width="551" alt="Screen Shot 2020-04-27 at 4 00 58 PM" src="https://user-images.githubusercontent.com/702990/80429177-5165ea80-88a0-11ea-938d-70375c6531dd.png">